draft-ietf-quic-transport.md

```diff
@@ -2391,13 +2391,17 @@ the packet other than the last 16 bytes for carrying data.
 
 ### Detecting a Stateless Reset
 
-An endpoint detects a potential stateless reset when a incoming packet
-with a short header either cannot be associated with a connection,
-cannot be decrypted, or is marked as a duplicate packet.  The endpoint
-then compares the last 16 bytes of the packet with the Stateless Reset
-Token provided by its peer, either in a NEW_CONNECTION_ID frame or
-the server's transport parameters.  If these values are identical,
-the endpoint MUST enter the draining period and not send any further
+An endpoint detects a potential stateless reset when an incoming packet either
+cannot be associated with a connection, cannot be decrypted, or is marked as a
+duplicate packet.  The endpoint MUST then compare the last 16 bytes of the
```

Comment:
Why MUST? SHOULD seems adequate here. A strawman argument is that an endpoint might choose to not use this if there are too many connections. I don't have a strong opposition, but I'd like to hear your rationale.

Reply:
Open an issue if you care.  I don't think that it's worth having wishy-washy SHOULDs here.

Reply:
I think that this can be a MUST, based on the understanding that processing of stateless resets sent by the peer is an option. Or are we requiring every endpoint to detect and handle stateless resets?